webapp/components/survey/SurveyForm/nodeDefs/components/types/nodeDefDate.js

```diff
@@ -0,0 +1,65 @@
+import './nodeDefDate.scss'
+
+import React, { useRef } from 'react'
+
+import PropTypes from 'prop-types'
+
+import * as Node from '@core/record/node'
+import * as DateUtils from '@core/dateUtils'
+
+import DateInput from 'react-input-date'
+
+const NodeDefDate = (props) => {
+  const { edit, entry, canEditRecord, nodeDef, nodes, readOnly, updateNode } = props
+
+  const inputField = useRef()
+
+  const node = entry ? nodes[0] : null
+
+  const nodeValue = Node.getValue(node, DateUtils.format(new Date(), 'dd/MM/yyyy'))
+
+  const handleChangeDateInput = (newDate) => {
+    const newDateSplited = newDate.split('/')
+    if (newDateSplited.length === 3) {
+      updateNode(nodeDef, node, newDate)
+    }
+  }
+
+  return (
+    <div className="survey-form__node-def-date">
+      <DateInput
```

Comment:
this component must be wrapped into a common component to be used also in other places